### PR TITLE
docs(blog): Documentation feedback

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Blog**
Adds a blog post providing information on documentation feedback

_Select the type of your PR_

* [ ] Typo/minor fix
* [x] New blog post (see the [README](https://github.com/strimzi/strimzi.github.io/#blog-posts) for the process)
* [ ] Other

### CODE REVIEW

File: _posts/2022-10-05-doc-feedback.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+date: 2022-10-05
+author: paul_mellor
+---
+
+Strimzi is supported by an extensive set of documentation. 
+If you want to know what Strimzi does, there's the [Overview](https://strimzi.io/docs/operators/latest/overview.html). 
+If you want to get up-and-running quickly, the [Quick Start guide](https://strimzi.io/docs/operators/latest/quickstart.html) can help. If you want to go deeper, we have specialized docs for [Deploying and Upgrading](https://strimzi.io/docs/operators/latest/deploying.html) or [Configuring](https://strimzi.io/docs/operators/latest/configuring.html) Strimzi.
+Kafka Bridge has its own [documentation](https://strimzi.io/docs/bridge/latest/) too. 
+
+But what don't we have have? And what do you want?
```
Comment (see-quick) [inline _posts/2022-10-05-doc-feedback.md:13]:
```suggestion
But what don't we have? And what do you want?
```
typo?

**Review by see-quick — APPROVED** [review]
Thanks for a such blog post :) Enjoyable reading Paul! 👍

**Review by scholzj — APPROVED** [review]

File: _posts/2022-10-05-doc-feedback.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## Adding to the documentation
+
+As well as feeding back, we really encourage the community to be involved with directly contributing to the documentation. If you spot an error in the docs or think it would benefit from a change or addition, you can open a documentation pull request.
+
+The documentation is written in asciidoc, a lightweight markup language similar to markdown.
+The [Strimzi Documentation Contributor Guide](https://strimzi.io/contributing/guide/#style-guide) shows how to make a change and push that change to the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator).
+
+You can also raise a [GitHub issue](https://github.com/strimzi/strimzi-kafka-operator/issues) against the documentation. Be sure to label it `documenation`.
+
+If you want to suggest a bigger change to the documentation before plunging in, you can always [raise a proposal](https://github.com/strimzi/proposals) for consideration.
```
Comment (ppatierno) [inline _posts/2022-10-05-doc-feedback.md:53]: I have never thought about using proposal for documentation changes. Isn't it too much?

**Review by ppatierno — APPROVED** [review]
LGTM. I left just one comment about documentation proposal.

File: _posts/2022-10-05-doc-feedback.md (hunk 2 above)
Comment (PaulRMellor) [reply _posts/2022-10-05-doc-feedback.md:53]: Possible. I've changed to _If you want to suggest a bigger change to the documentation before plunging in, you can always raise it with the community for consideration._

File: _posts/2022-10-05-doc-feedback.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+We might not know about them.
+
+## Adding to the documentation
+
+As well as feeding back, we really encourage the community to be involved with directly contributing to the documentation. If you spot an error in the docs or think it would benefit from a change or addition, you can open a documentation pull request.
+
+The documentation is written in asciidoc, a lightweight markup language similar to markdown.
+The [Strimzi Documentation Contributor Guide](https://strimzi.io/contributing/guide/#style-guide) shows how to make a change and push that change to the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator).
+
+You can also raise a [GitHub issue](https://github.com/strimzi/strimzi-kafka-operator/issues) against the documentation. Be sure to label it `documenation`.
```
Comment (tombentley) [inline _posts/2022-10-05-doc-feedback.md:51]:
```suggestion
You can also raise a [GitHub issue](https://github.com/strimzi/strimzi-kafka-operator/issues) against the documentation. Be sure to label it `documentation`.
```

**Review by tombentley — APPROVED** [review]

**Pull request merged**